Cargo.toml

```diff
@@ -22,7 +22,7 @@ byteorder = { version = "1", default-features = false }
 serde = { version = "1.0", default-features = false, features = ["derive"], optional = true }
 mint = "^0.5.4"
 bitflags = "1"
-num-traits = "0.2.15"
+num-traits = { version = "0.2.15", default-features = false}
```

Comment:
It would be better to enable `libm` feature too as it will enable f32 and f64 functions

Reply:
See #27